### ISSUE TITLE
Multi-line type signatures does not highlight correctly

### ISSUE THREAD

**Issue by ahstro** [ElmCast/elm-vim#156]
Notice that the multi-line `foo` is lacking the yellow color that the single-line `foo` has:

![highlight](https://user-images.githubusercontent.com/3090888/38927304-9877efdc-4305-11e8-9936-9ef3961d4ea5.png)

**Comment by leohaskell** [comment]
Indeed, currently, the highlight is very broken for functions having multi-line type signatures:

<img width="467" alt="Screenshot 2019-08-13 at 11 29 40" src="https://user-images.githubusercontent.com/3736978/62926700-b1256f80-bdbd-11e9-8dee-995fd488b8a2.png">
